src/internal/DummyInput.js

```diff
@@ -3,28 +3,48 @@
 import React from 'react';
 import { Input } from '../primitives';
 
-export default (props: any) => (
```

Comment:
A safer way to do this would be to pull out the props that we know could be included (for the `Animated` components) and exclude them. That way we don't need to maintain a white-list of all props.

e.g

```js
export default ({ in, out, ...props }: any) => {
```

Reply:
Had considered that option but is it only these props for Animated components that we would ever need to exclude?

Reply:
Disregard my last question. I have discussed with @gwyneplaine and have better context now on this one.